learn/glossary.md

```diff
@@ -24,15 +24,30 @@ While future drafts may introduce new behavior or changes to existing behavior,
 
 The current list of drafts can be found [here](https://json-schema.org/specification-links.html#published-drafts).
 
+### keyword
+
+A property appearing within a [schema](#schema) object.
+
+The [JSON Schema specification](https://json-schema.org/specification.html) defines behavior for a large library of keywords which can be used to describe [instances](#instance).
+
 ### instance
 
-A piece of JSON data which is to be validated by a [schema](#schema).
+A piece of JSON data which is to be described by a [schema](#schema).
 
-JSON Schema can be used to validate JSON values of any type (as well as values from many JSON-like formats which can be reasonably represented as JSON).
+JSON Schema can be used to describe JSON values of any type (as well as values from many JSON-like formats which can be reasonably represented as JSON).
 
 The JSON Schema specification makes no broad assumptions about the structure of instances themselves beyond those of the JSON specification itself.
 In particular it does not reserve any properties within a JSON object for its own use, or require parsers of JSON to support features beyond those already mandated of JSON implementations.
 
+### meta-schema
+
+A [schema](#schema) which is itself intended to describe other *schemas*.
+
+JSON Schema defines a language for describing any [instance](#instance) using a schema written in JSON.
+Since schemas are themselves JSON values, they may be opaquely treated as *instances*, and therefore described by other schemas.
```

Comment:
Actually it's not entirely opaque, §9.3.3 notes that in some circumstances an alternate way of handling schema-instances SHOULD be implemented.  I'm not quite sure how that should be dealt with as we don't want a huge tangent on an obscure case, but it should be acknowledged that meta-schema validation can be more complex.

Reply:
Are you happy with deleting the word `opaquely` and just saying `may also be treated as instances`? That matches the language from [§4.3](https://json-schema.org/draft/2020-12/json-schema-core.html#name-json-schema-documents).

Reply:
@Julian yeah that would work, thanks!  Better than too much detail on the basic concept.  I'll bonk the approve button again when I see that change, but feel free to consider it pre-approved - everything else is great.

Reply:
Great, done. Thanks.